setup.cfg

```diff
@@ -0,0 +1,17 @@
+[bdist_wheel]
+universal = 1
+
+[flake8]
+max-line-length = 100
+inline-quotes = double
+
+[isort]
```

Comment:
I'm not convinced by putting isort configuration in this project - pep8 and pyflake are quite standard and add value, isort seems quite specific.

Reply:
it is used by[ flake8-isort ](https://pypi.org/project/flake8-isort/)